core_lib/src/structure/layerbitmap.cpp

```diff
@@ -46,6 +44,18 @@ BitmapImage* LayerBitmap::getLastBitmapImageAtFrame(int frameNumber, int increme
     return static_cast<BitmapImage*>(getLastKeyFrameAtPosition(frameNumber + increment));
 }
 
+void LayerBitmap::repositionFrame(QPoint transform, int frame)
+{
+    BitmapImage* image = getBitmapImageAtFrame(frame);
+    image->moveTopLeft(transform);
+}
+
+QRect LayerBitmap::getFrameBounds(int frame)
+{
+    BitmapImage* image = getBitmapImageAtFrame(frame);
+    return image->bounds();
+}
+
```

Comment:
Missed this in the review...

repositionFrame is misleading, as I'd expect the implementation to reposition a keyframe and not the content of the image. A better name would be repositionImage. The parameter called transform is also misleading, as transform is often related to QTransform. Call it pos or point to relate it to a qpoint.